### PR TITLE
backlog in frontend

### PR DESCRIPTION
**Pull request by baxtertyler**
backlog.js - new file for front end component of the task backlog
backlog.css - css for react for backlog
home.js - HISTORY is not a button rather than just a word
App.js - added route for backlog
*minor changes on other files

### CODE REVIEW

**Comment by lukewaltz** [comment]
Looks good. make sure you have your backend .env file in your .gitignore.

File: packages/express-backend/.env
Comment (lukewaltz) [inline packages/express-backend/.env:1]: we gotta get this out of github

File: packages/react-frontend/src/pages/backlog.css
Comment (lukewaltz) [inline packages/react-frontend/src/pages/backlog.css:1]: was this all you? or did you just borrow most of the styling from stuff kylie wrote. either way, kudos.

File: packages/react-frontend/src/pages/backlog.js
Comment (lukewaltz) [inline packages/react-frontend/src/pages/backlog.js:1]: LGTM!

File: packages/react-frontend/src/pages/backlog.css
Comment (kylieodonnell) [reply packages/react-frontend/src/pages/backlog.css:1]: saw this in lab, and looks pretty freaking cool!!

File: packages/react-frontend/src/App.js
Comment (kylieodonnell) [inline packages/react-frontend/src/App.js:1]: yay, history is implemented!

File: packages/react-frontend/src/pages/home.js
Comment (kylieodonnell) [inline packages/react-frontend/src/pages/home.js:1]: this is great. excited to know this works.

**Comment by yarecryb** [comment]
LGTM!

**Pull request merged**